clients/tango-cli.py

```
@@ -12,6 +12,10 @@
 import argparse
 import sys
 import os
+import time
+from datetime import datetime
+import json
```

Ruff

Redefinition of unused `json` from line 10

Remove definition: `json`

(F811)